service/src/test/java/bio/terra/profile/service/spendreporting/azure/AzureSpendReportingServiceUnitTest.java

```diff
@@ -140,7 +140,8 @@ void testGetBillingProfileSpendData_K8sResourceGroupExists_Success() {
             SpendCategoryType.STORAGE,
             from,
             to);
-    when(mockQueryResultMapper.mapQueryResult(queryResult2, SpendCategoryType.COMPUTE, from, to))
+    when(mockQueryResultMapper.mapQueryResult(
+            queryResult2, SpendCategoryType.WORKSPACE_INFRASTRUCTURE, from, to))
```

Comment:
Why is Storage going to WORKSPACE_INFRASTRUCTURE?

Reply:
QueryResult queryResult2 = mock(QueryResult.class);
    when(queryResult2.rows()).thenReturn(List.of(QUERY_RESULT_DATA.get(STORAGE_RESOURCE_TYPE)));
    when(response2.getValue()).thenReturn(queryResult2);

Reply:
The WORKSPACE_INFRASTRUCTURE is a category for all resources under K8s node resource group. There might be different type of resources in that resource group. For instance - managed identity, virtual machine scale set, route table, etc. Agree that in this case it's not really successful mocking (not sure that storage can be part of K8s node resource group). I will adjust test data and mocking to be more consistent.

Reply:
Thanks, this helps. I noticed a place below where it would be good to also rename the variable appropriately.